src/sentry/api/serializers/models/project.py

```diff
@@ -236,6 +236,28 @@ def serialize(self, obj, attrs, user):
 
 
 class ProjectSummarySerializer(ProjectWithTeamSerializer):
+    def get_attrs(self, item_list, user):
```

Comment:
Do we want this only under `ProjectSummarySerializer`? Or should it just be on the base `ProjectSerializer` ?

Reply:
We only need it here for now, not sure if it's better to just put it on the base serializer, given that the same property already exists on the DetailedProjectSerializer anyway

Reply:
Wait, you're right. If it's already on DetailedProjectSerializer, let's just move it to the base class.